ndmg/scripts/ndmg_dwi_pipeline.py

```diff
@@ -280,6 +277,9 @@ def ndmg_dwi_worker(
     seeds = track.build_seed_list(reg.wm_gm_int_in_dwi, np.eye(4), dens=int(seeds))
     print("Using " + str(len(seeds)) + " seeds...")
 
+    #create location to save tensor qa
+    qa_tensor = namer.name_derivative(namer.dirs["qa"]["tensor"],"Tractography_Model_Peak_Directions.png")
```

Comment:
rename this so that it's more clear that it's a string link to an output file
note that I'm deprecating `namer` this week, but I'll fix this line myself when I do.